Library/Formula/ice.rb

```diff
@@ -32,7 +32,10 @@ def install
         --enable-cxx
       ]
 
-      args << "--enable-java" if build.with? "java"
+      if build.with? "java"
+        args << "--enable-java"
+        inreplace "dist/Makefile.in", "@JAVACFLAGS@",  "@JAVACFLAGS@ -source 1.7 -target 1.7"
```

Comment:
Can you add a comment about having submitted a ticket upstream, so it doesn't end up as a floating patch?